README.md

```diff
@@ -1,43 +1,46 @@
+- **SQL database** for surveys and other tabular data.
 
-### 📦 App stack
+### 🔐 Access Control
 
-Deploy software easily with [CapRover](https://caprover.com/).
+Secure login system to control who can access your applications.
+
+### 📦 App stack
 
 - **Windmill**: Sync from outside data providers, schedule your data workflows, and run other scripts.
 - **Superset**: Visualize and explore structured/tabular data with interactive dashboards.
 - **Filebrowser**: Easily view and create share links for your media (pictures, videos...).
 - **GuardianConnector Explorer**: Unified view of tabular, media, and maps.
 - **CoMapeo Archive Server**: A CoMapeo archive server that synchronizes with CoMapeo devices.
-- ...plus you can install any other web-based software you want.
+- ...plus install any other web applications you need using [CapRover](https://caprover.com/).
+
+## Deployment Guide
 
-## High-level steps to spawn a new stack
+High-level steps to spawn a new stack
 
-1. Deploy a VM.
-    - [`azure/README.md`](azure/README.md) describes setting this up on Azure.
-    - [`digitalocean-vm/README.md`](digitalocean-vm/README.md) describes setting up a VM with CapRover included on DigitalOcean.
-    - With the right amount of knowledge, you can follow the above guides and/or CapRover's own [Getting Started](https://caprover.com/docs/get-started.html) guide to figure out how to deploy a VM on your hosting platform of choice.
+1. 🖥️ Choose your cloud provider and deploy a VM running CapRover.
+    - **[Azure Setup Guide](azure/README.md)**
+    - **[DigitalOcean Setup Guide](digitalocean-vm/README.md)**
+    - With the right amount of knowledge, you can follow CapRover's own [Getting Started](https://caprover.com/docs/get-started.html) guide to figure out how to deploy a VM on your hosting platform of choice.
 
-2. Create your data warehouse: file storage & a SQL database.
+2. 📂 Create your data warehouse: file storage & a SQL database.
     - The Azure deployment can automatically create an Azure Storage Account and Files share for you, or you can use an existing one. If you are storing files on the VM directly, you can skip Azure Files entirely.
-2. If you plan on using auth0, set up an **auth0 tenant** with a user authorization workflow and client applications to help secure access to the apps you will deploy. See [`auth0/README.md`](auth0/README.md)
+3. 🔐 If you plan on using auth0, set up an **auth0 tenant** with a user authorization workflow and client applications to help secure access to the apps you will deploy. See [`auth0/README.md`](auth0/README.md)
 
-3. Install the app stack by following [`caprover/README.md`](https://github.com/ConservationMetrics/gc-forge/blob/main/caprover/README.md).
-4. Set up data pipelines & other scripts to run in Windmill.
+4. 📦 Install the app stack by following [`caprover/README.md`](https://github.com/ConservationMetrics/gc-forge/blob/main/caprover/README.md).
```

Comment:
Just to confirm - `caprover/README.md` is yet to be ported over, and these links to the `gc-forge` readme will change when done so?